### ISSUE TITLE
unable to update the global install of aurelia-cli

### ISSUE THREAD

**Issue by chrsmrtn-** [aurelia/cli#386]
**I'm submitting a bug report**

* **Library Version:**
0.21.0

**Please tell us about your environment:**
* **Operating System:**
Windows 10 Anniversary update

* **Hardware:**
Surface Book (no longer reproduces on my Dell laptop.

* **Node Version:**
6.2.0

* **NPM Version:**
3.10.8

* **Browser:**
N/A

* **Language:**
N/A

**Current behavior:**
I'm having issues updating my global installation of Aurelia-cli.

I ran: `npm uninstall -g aurelia-cli`
Then I ran `npm install -g aurelia-cli@0.21.0` to make sure I got the latest version
When I run `au -v` the version reported is not what i'd expect.  Here is the command line output from when I installed the Aurelia-cli 0.20.0 and tested the verion

This issue still exists if you close and open a new command prompt.

Here is the full output from my command prompt:

```
C:\Users\chrsm\Documents\Visual Studio Code>npm install -g aurelia-cli@0.21.0
C:\Users\chrsm\AppData\Roaming\npm\au -> C:\Users\chrsm\AppData\Roaming\npm\node_modules\aurelia-cli\bin\aurelia-cli.js
C:\Users\chrsm\AppData\Roaming\npm\aurelia -> C:\Users\chrsm\AppData\Roaming\npm\node_modules\aurelia-cli\bin\aurelia-cli.js
C:\Users\chrsm\AppData\Roaming\npm
`-- aurelia-cli@0.21.0


C:\Users\chrsm\Documents\Visual Studio Code>au -v
0.19.0

C:\Users\chrsm\Documents\Visual Studio Code>cd C:\Users\chrsm\AppData\Roaming\npm

C:\Users\chrsm\AppData\Roaming\npm>au -v
0.21.0
```

**Expected/desired behavior:**

* **What is the expected behavior?**
I'd expect `au -v` to report the right version that I just installed, 0.21.0

* **What is the motivation / use case for changing the behavior?**
I would like to update so I can take advantage of the improvements when I create a new Aurelia project or update an existing Aurelia project with updated script files like the test script examples or au commands.

**Comment by AStoker** [comment]
Run 
`npm uninstall -g aurelia-cli au`
`npm cache clean`
`npm install -g aurelia-cli`

Sometimes npm leaves the `au` sucker in there.

**Comment by chrsmrtn-** [comment]
ran the commands and the issue still persists.  Used `...` to help shrink the console output.  I left the first and last subline output for each command

```
C:\Users\chrsm\Documents\Visual Studio Code>npm uninstall -g aurelia-cli
- aurelia-pal@1.0.0 node_modules\aurelia-cli\node_modules\aurelia-pal
...
- aurelia-cli@0.21.0 node_modules\aurelia-cli

C:\Users\chrsm\Documents\Visual Studio Code>npm cache clean

C:\Users\chrsm\Documents\Visual Studio Code>npm install -g aurelia-cli
C:\Users\chrsm\AppData\Roaming\npm\au -> C:\Users\chrsm\AppData\Roaming\npm\node_modules\aurelia-cli\bin\aurelia-cli.js
C:\Users\chrsm\AppData\Roaming\npm\aurelia -> C:\Users\chrsm\AppData\Roaming\npm\node_modules\aurelia-cli\bin\aurelia-cli.js
C:\Users\chrsm\AppData\Roaming\npm
`-- aurelia-cli@0.21.0
...
    `-- write-file-atomic@1.2.0


C:\Users\chrsm\Documents\Visual Studio Code>au -v
0.19.0
```

**Comment by chrsmrtn-** [comment]
I also ran the uninstall and then closed the command prompt. Then from a fresh command prompt I ran `npm -g cache clear`.  Then I ran the global install of aurelia-cli.  Finally ran `au -v` and still got the unexpected output of 0.19.0

**Comment by corneliutusnea** [comment]
@chrsmrtn- 
1. Are you running that from an Admin command prompt?
2. What version of npm do you run? Should be  a 3.10.x.
Also make sure you run this:
https://www.npmjs.com/package/npm-windows-upgrade
Lastly, you migth want to delete your cache folders manually. Npm has sometimes issues with packages:
Try to delete aurelia-cli from C:\Program Files\nodejs\node_modules
and everythign from 
C:\Users\chrsm\AppData\Roaming\npm

**Comment by AStoker** [comment]
@Chrsmrtn You missed the whole command. 
Specifically, run `npm uninstall -g aurelia-cli au`
We're uninstalling `aurelia-cli` *AND* `au`.
After you do that, run the rest of the commands I gave you.

**Comment by chrsmrtn-** [comment]
@AStoker Nice catch! I made sure to in `npm uninstall -g aurelia-cli au` as the first command and got the same result :(

@corneliutusnea I am running with local user permissions as everything is being installed in my AppData folder.  When running with the -g flag, nothing gets installed in C:\Program Files\nodejs\node_modules. Even when running as Administrator. I am using npm version 3.10.8.  I uninstalled Aurelia-cli with my user account, then attempted to install with Administrator priviledges.

Now before attempting the install with Administrator priviledges I did a version check.  And found something odd.

So in my environment I globally installed Aurelia-cli using my user with regular permissions.  When I run `au -v` it reports 0.19.0.  However when I run `au -v` from a command prompt with Administrator privileges it reports, 0.21.0  Both commands are being referenced from `C:\Users\chrsm\AppData\Roaming\npm\` folder.

See attached image
![aurelia-cli-versions](https://cloud.githubusercontent.com/assets/5657979/19697252/84743bae-9a9f-11e6-9168-9a1a56b64fcd.PNG)

**Comment by chrsmrtn-** [comment]
@AStoker Okay this is odd.  I ran the commands you recommended on my other dev machine, my Dell Precision M4800, and the version is properly being reported.  All of this done under my local user account.

This issue has reproduced on this machine before.  But now it's not.  The screen captures I provided here and have been testing on thus far is my Surface Book.

I did another test.  I ran the uninstall and cache clear commands in a command prompt.  Then closed and opened a fresh one.  I then ran the install command and restarted the Surface Book.  The issue is still reproducing on the Surface Book.

Going to update the main ticket with the hardware info.

**Comment by AStoker** [comment]
It might be possible that at one point your ran `npm install aurelia-cli` (no `-g` command) in a location and you've been running your install commands globally.
Run the uninstall command I gave you earlier (au and aurelia-cli), then try `au -v` and see what you get. You should get some error saying `au` is undefined.

**Comment by chrsmrtn-** [comment]
Yep, that was it!  Turns out on my SurfaceBook I ran `au install aurelia-cli` at `C:\Users\chrsm`.  This mean that was the one that gets picked up even if you run `au -v` from the `C:\Users\chrsm\Documents\Visual Studio Code\` directory.  So while it was indeed using the globally installed au exe in my AppData for my profile, the execution version was being scoped down to the one installed at `C:\Users\chrsm`

Thanks for all your help!  Cheers!
